scripts/setup-mel-builddir

```diff
@@ -310,3 +315,7 @@ MEL_DISTRO="${MEL_DISTRO:-mel}"
 if [ -e $BUILDDIR/conf/local.conf ]; then
     sed -i -e "s/^DISTRO.*/DISTRO = \'$MEL_DISTRO\'/" $BUILDDIR/conf/local.conf
 fi
+
+if [ "$toasterconfig" = "yes" ]; then
+    $layerdir/scripts/toaster_configuration "$BUILDDIR"
+fi
```

Comment:
This isn't an unreasonable approach. Of course, in the other mechanism, automatically running it, it added overhead to every setup environment execution, but with this one, we'd have to update the docs. I'd suggest running it by kathy and possibly @challinan